sled-agent/src/services.rs

```diff
@@ -132,10 +132,10 @@ const IPV6_UNSPECIFIED: IpAddr = IpAddr::V6(Ipv6Addr::UNSPECIFIED);
 const COCKROACH: &str = "/opt/oxide/cockroachdb/bin/cockroach";
 
 const CLICKHOUSE_SERVER_BINARY: &str =
-    "/opt/oxide//opt/oxide/clickhouse_server/clickhouse";
+    "/opt/oxide/clickhouse_server/clickhouse";
```

Comment:
Are these three binaries actually the same binary but put in different paths in the different zone types? (If so that seems fine but maybe comment-worthy. If not, can we give the binaries different names?)

Reply:
Done in [3a6b659](https://github.com/oxidecomputer/omicron/pull/6894/commits/3a6b6592f1a8a02816ab1ac7495b98e56164b9b4)